x/amm/client/cli/tx_create_pool.go

```diff
@@ -18,7 +18,7 @@ func CmdCreatePool() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "create-pool [weights] [initial-deposit] [swap-fee] [exit-fee]",
 		Short:   "create a new pool and provide the liquidity to it",
-		Example: `elysd tx amm create-pool 100ueden,100uelys 1000000ueden,1000000uelys 0.00 0.00  --from=treasury --keyring-backend=test --chain-id=elystestnet-1 --yes --gas=1000000`,
+		Example: `elysd tx amm create-pool 100uusdc,100uatom 100000000000uusdc,100000000000uatom 0.00 0.00  --from=treasury --keyring-backend=test --chain-id=elystestnet-1 --yes --gas=1000000`,
```

Comment:
I think 100uatom,100uusdc for sorting, not 100uusdc,100uatom

Reply:
@jelysn any reason why it must be in this order?